data/backend/authentication/urls.py

```diff
@@ -0,0 +1,23 @@
+from . import views
+from .views import register, RegistrationRequestedView, profile
+from django.urls import include, path, re_path
+
+
+urlpatterns = [
+    path("login/", views.LoginView.as_view(), name="login"),
+    path("login.json/", views.LoginView.as_view(), name="login_json"),
```

Comment:
Why would we want a filename with a trailing slash? And why do we want two differently named routes resolving to the same view controller? Why not just using the `login` route?

Reply:
I've removed the classical django register/login, and added a parcel proxy path for `/accounts` and `/processes` so that we can test register/login in dev mode